node/src/daemon/setup_reporter.rs

```diff
@@ -999,13 +1014,19 @@ impl ValueRetriever for Neighbors {
 
     fn computed_default(
         &self,
-        _bootstrapper_config: &BootstrapperConfig,
+        bootstrapper_config: &BootstrapperConfig,
```

Comment:
`node_descriptors_to_neighbors()` now needs a `CryptDE` because it can no longer access the static one.